discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -378,6 +379,31 @@ def fetch_existing_entities(session: Session, entities_to_fetch: EntitiesToFetch
             for repost in reposts
         }
 
+    # SUBSCRIPTIONS
+    if entities_to_fetch[EntityType.SUBSCRIPTION]:
+        subscriptions_to_fetch: Set[Tuple] = entities_to_fetch[EntityType.SUBSCRIPTION]
+        and_queries = []
+        for subscription_to_fetch in subscriptions_to_fetch:
+            user_id = subscription_to_fetch[0]
+            # subscriptions does not need entity type in subscription_to_fetch[1]
+            entity_id = subscription_to_fetch[2]
+            and_queries.append(
```

Comment:
my fault for calling these and_queries but they should be or, right?

Reply:
`and_queries` makes sense to me; its an array of `and` queries that are later joined together with `or`

Reply:
ooh yea true haha